scripts/test/directtools/DirectToolsTest.py

```diff
@@ -0,0 +1,225 @@
+from __future__ import (absolute_import, division, print_function)
+
+import directtools
+from mantid.api import mtd
+from mantid.simpleapi import LoadILLTOF, CreateSampleWorkspace, CreateWorkspace
+import matplotlib
+import numpy
+import numpy.testing
+import testhelpers
+import unittest
+
+
+class DirectTest(unittest.TestCase):
+
+    def tearDown(self):
+        mtd.clear()
+
+    def test_box2D(self):
```

Comment:
There's a lot of repeated code in this test. If each group of lines is testing something different, would it be best to split them into their own smaller tests?

( similarly `test_nanminmax`, `test_plotconstE`, `test_plotconstQ`, `test_plotprofiles`)